idd/Energy+.idd.in

```diff
@@ -61443,7 +61449,7 @@ AirConditioner:VariableRefrigerantFlow:FluidTemperatureControl,
        \note OnDemand to defrost only when necessary
   A10, \field Defrost Energy Input Ratio Modifier Function of Temperature Curve Name
        \type object-list
-       \object-list UnivariateFunctions
+       \object-list BivariateFunctions
```

Comment:
OK, so everything was implemented properly in the code, but IDF-Editor wouldn't show the right curves.  Got it.